app/models/training/question.rb

```diff
@@ -30,16 +30,16 @@ def to_partial_path
 
     # @return [Boolean]
     def multi_select?
-      confidence_question? || opinion_question? ? false : answer.multi_select?
+      confidence_question? || feedback_question? ? false : answer.multi_select?
     end
 
-    def opinion_question?
-      page_type == 'opinion'
+    def feedback_question?
```

Comment:
This method definition is redundant because you have already defined it in the `ContentTypes` module 

```ruby
  # @return [Boolean]
  def feedback_question?
    page_type.eql?('feedback')
  end
```